src/is_subset.jl

```diff
@@ -195,6 +195,69 @@ function ⊆(P::AbstractPolytope{N}, S::LazySet{N}, witness::Bool=false
     end
 end
 
+"""
+    ⊆(S::LazySet{N},
+      P::Union{AbstractPolytope{N}, HPolyhedron{N}, HalfSpace{N}},
+      witness::Bool=false
+     )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+
+Check whether a convex set is contained in a polyhedron, and if not, optionally
+compute a witness.
+
+### Input
+
+- `S` -- inner convex set
+- `P` -- outer polyhedron (including a half-space)
+- `witness` -- (optional, default: `false`) compute a witness if activated
+
+### Output
+
+* If `witness` option is deactivated: `true` iff ``S ⊆ P``
+* If `witness` option is activated:
+  * `(true, [])` iff ``S ⊆ P``
+  * `(false, v)` iff ``S ⊈ P`` and ``v ∈ P \\setminus S``
+
+### Algorithm
+
+Since ``S`` is convex, we can compare the support function of ``S`` and ``P`` in
+each direction of the constraints of ``P``.
+
+For witness generation, we use the support vector in the first direction where
+the above check fails.
+"""
+function ⊆(S::LazySet{N},
+           P::Union{AbstractPolytope{N}, HPolyhedron{N}, HalfSpace{N}},
+           witness::Bool=false
+          )::Union{Bool, Tuple{Bool, Vector{N}}} where {N<:Real}
+    for H in constraints_list(P)
+        if ρ(H.a, S) > H.b
```

Comment:
```suggestion
        if _geq(ρ(H.a, S), H.b)
```

Reply:
actually i'm not sure, if you want strict inequality we can leave it as is

Reply:
Yes, I want to check the negation of `<=`.